### ISSUE TITLE
Embeddable widget not accessible

### ISSUE THREAD

**Issue by mitorious** [Automattic/WP-Job-Manager#2144]
Hi, I am using the embeddable jobs widget to put our posts on another web page. Sadly this causes the page to fail it's WCAG 2.1 accessibility target.

The error it throws is: 

Requirement: Links that open in a new tab or a new window should either have an "aria-label" attribute or a screen-reader only element explaining to screen-readers that this opens in a new tab.

This doesn't seem to be possible to fix through the back end. can this be added in please?

Thanks

**Comment by alexsanford** [comment]
Thank you for the report! I've added an issue to the Embeddable Job Widget repository and added it to the queue.

In the meantime, it looks like the offending links are in the `templates/content-embeddable-widget-job_listing.php` template file. As a workaround, it's possible for you to [override this file](https://wpjobmanager.com/document/template-overrides/) and add the required attribute or screenreader text and improve the accessibility.

**Comment by mitorious** [comment]
Thanks! I'll take a look.
